src/support.js

```diff
@@ -638,7 +640,9 @@ function addFunction(func, sig) {
     }
   }
   throw 'Finished up all reserved function pointers. Use a higher value for RESERVED_FUNCTION_POINTERS.';
-#else
+
+#else // EMULATED_FUNCTION_POINTERS == 0
```

Comment:
Isn't this `EMULATED_FUNCTION_POINTERS != 0`?

Reply:
I am repeating the clause form the "if" here, and again at the end.  The idea is that you can match all three visually.  Do you think inverting the logic would make it more clear?  What about the comment on the final endif?

Reply:
Oh, I see what you mean. Nevermind then.